pkg/database/mongoRepository.go

```diff
@@ -94,8 +95,10 @@ func (m *MongoRepository) GetAllMentalHealthLogsByDate(ctx context.Context, user
 	return toReturn, err
 }
 
-func (m *MongoRepository) GetOverallScore(ctx context.Context, userID int64) (float64, error) {
+func (m *MongoRepository) GetOverallScore(ctx context.Context, userID int64) (int32, error) {
 	logs, err := m.GetAllMentalHealthLogs(ctx, userID)
+	log.Printf("Logs fetched for user (ID = %v\n): %v\n", userID, logs)
+
 	if err != nil {
 		log.Fatal("cannot get mental health logs for user: ", err)
```

Comment:
Use the injected zap logger stored in the struct, not the stdlib logger. (This is the case in many points here, resolve them all please)

Reply:
Changed all the instances of this!